core/assets/sounds/cues/countInEn/data.json

```diff
@@ -0,0 +1,188 @@
+{
+  "gameID": "countInEn",
+  "gameName": "Count-Ins (English)",
+  "cues": [
+    {
+      "id": "countInEn/silence",
+      "duration": 1,
+      "canAlterDuration": true,
+      "loops": false,
+      "name": "silence"
+    },
+    {
+      "id": "countInEn/cowbell",
+      "duration": 1,
+      "name": "cowbell"
+    },
+    {
+      "id": "countInEn/one",
+      "duration": 1,
+      "name": "one"
+    },
+    {
+      "id": "countInEn/two",
+      "duration": 1,
+      "name": "two"
+    },
+    {
+      "id": "countInEn/three",
+      "duration": 1,
+      "name": "three"
+    },
+    {
+      "id": "countInEn/four",
+      "duration": 1,
+      "name": "four"
+    },
+    {
+      "id": "countInEn/and",
+      "duration": 0.5,
+      "name": "and"
+    },
+    {
+      "id": "countInEn/go1",
+      "duration": 0.5,
+      "name": "go - 1"
+    },
+    {
+      "id": "countInEn/go2",
+      "duration": 0.5,
+      "name": "go - 2"
+    }
+  ],
+  "patterns": [
+    {
+      "id": "countInEn_2BeatCountIn",
+      "deprecatedIDs": [
+        "countinen_2BeatCountIn"
+      ],
+      "name": "2-beat count-in",
+      "cues": [
+        {
+          "id": "countInEn/three",
+          "beat": 0
+        },
+        {
+          "id": "countInEn/four",
+          "beat": 1
+        }
+      ]
+    },
+    {
+      "id": "countInEn_4BeatCountIn",
+      "deprecatedIDs": [
+        "countinen_4BeatCountIn"
+      ],
+      "name": "4-beat count-in",
+      "cues": [
+        {
+          "id": "countInEn/one",
+          "beat": 0
+        },
+        {
+          "id": "countInEn/two",
+          "beat": 1
+        },
+        {
+          "id": "countInEn/three",
+          "beat": 2
+        },
+        {
+          "id": "countInEn/four",
+          "beat": 3
+        }
+      ]
+    },
+    {
+      "id": "countInEn_8BeatCountIn",
+      "deprecatedIDs": [
+        "countinen_8BeatCountIn"
+      ],
+      "name": "8-beat count-in",
+      "cues": [
+        {
+          "id": "countInEn/one",
+          "beat": 0
+        },
+        {
+          "id": "countInEn/two",
+          "beat": 2
+        },
+        {
+          "id": "countInEn/one",
+          "beat": 4
+        },
+        {
+          "id": "countInEn/two",
+          "beat": 5
+        },
+        {
+          "id": "countInEn/three",
+          "beat": 6
+        },
+        {
+          "id": "countInEn/four",
+          "beat": 7
+        }
+      ]
+    },
+    {
+      "id": "countInEn_4BeatCountInCowbell",
+      "deprecatedIDs": [
+        "countinen_4BeatCountInCowbell"

```

Comment:
You clearly didn't test this with deprecations. This is not a valid deprecated ID.